tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -436,6 +436,8 @@ install_virtiofsd() {
 
 # Install static nydus asset
 install_nydus() {
+	[ "${ARCH}" == "aarch64" ] && ARCH=arm64
+
 	install_cached_tarball_component \
 		"nydus" \
 		"${jenkins_url}/job/kata-containers-main-nydus-$(uname -m)/${cached_artifacts_path}" \
```

Comment:
I wonder if it's worth updating the name of the nydus cache job from kata-containers-main-nydus-aarch64 to kata-containers-main-nydus-arm64 to match with the nydus release?

Reply:
That's a good point, but I think it'd be then incosistent with the rest of the jobs.

I have no preference here, as I think both cases are not the optimal ones.  WDYT?

Reply:
I'm fine with it as it currently is, as you say neither solutions are perfect given the inconsistency in nydus, so I thought the comment was worth discussing about whether we align with nydus, or our other jobs, but I wasn't convinced that would be better!